signer/tests/integration/transaction_coordinator.rs

```diff
@@ -1384,6 +1444,15 @@ async fn sign_bitcoin_transaction() {
     faucet.send_to(50_000_000, &depositor.address);
     faucet.generate_blocks(1);
 
+    wait_for_signers(&signers).await;
+    tokio::time::sleep(Duration::from_secs(2)).await;
```

Comment:
just curious, what was this (and the repetitions below) sleep needed for (since you `wait_for_signers` above)?

Reply:
I was getting errors without them. Let me try to remove them, I might not need them anymore.